frontend/packages/console-dynamic-plugin-sdk/src/app/components/factory/ListPage/ListPageBody.tsx

```diff
@@ -0,0 +1,8 @@
+import * as React from 'react';
+import '../../../../styles/common.scss';
```

Comment:
Not blocking: at some point we should investigate using aliases so we don't need to use deep relative paths such as `../../../../`

Reply:
used the alias, thank you :-)